snippets/ModalDialog.md

```diff
@@ -0,0 +1,133 @@
+### Modal/Dialog
+
+Renders a dialog component with modal.
+
+This is a simple React ModalDialog Component.It owns 4 props: `show`、`close`、`title` and `clickModal2Hide`.
+
+* `show` prop determines the initial state of the dialog(show/hide).It can influence the `display` css property of `modal` class div.E
```

Comment:
I'm not seeing where this is used anymore( Find on `this.props` only shows the click to hide).

If this were to be used we'd need another prop for initial content, which is doable, but I think instead just calling to show on initial render of the app with content probably works better. Suggest removing it.

```suggestion

```